File: LinkedList/introducation.cpp
#include <iostream>
using namespace std;

struct Node {
    public:
    int data;
    Node* next;

    public:
    Node(int data1, Node* next1){
        data = data1;
        next = next1;
    }


    public:
    Node(int data1){
        data = data1;
        next = nullptr;
    }
};

int main(){

    Node* y = new Node(55,nullptr);
    cout<<y->data;
    return 0;
}

File: LinkedList/Introducation.java
class Node {
    int data;
    Node next;

    Node(int data1, Node next1) {
        data = data1;
        next = next1;
    }

    Node(int data1) {
        data = data1;
        next = null;
    }
};

public class Introducation {

    public static void main(String[] args) {

        Node y = new Node(3, null);
        System.err.println(y.data);
    }
};